src/react-apps/applications/service-development/src/features/deploy/containers/deployToTestContainer.tsx

```diff
@@ -0,0 +1,178 @@
+      if (repoStatus.contentStatus.length > 0) {
+        return 'ahead';
+      } else if (repoStatus.behindBy > 0) {
+        return 'behind';
+      } else {
+        return 'ready';
+      }
+    } else {
+      return null;
+    }
+  }
+
+  public returnMasterRepoAndDeployInSync = (env: string, masterRepoStatus: any, deploymentList: any): any => {
+    const image = deploymentList[env].items[0].spec.template.spec.containers[0].image;
+    const imageTag = image.split(':')[1];
+    if (masterRepoStatus !== null && masterRepoStatus.commit.id === imageTag) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  public render() {
+    const { classes, language } = this.props;
+    const { deploySuccess } = this.state;
+
+    return (
+      <React.Fragment>
+        <div className={classes.mainLayout}>
+          <Grid container={true} justify='center' className={classes.mainGridStyle}>
+            <Grid item={true} xs={11} sm={10} md={11}>
+              <Typography variant='h1' style={{ fontWeight: 400 }}>
+                {getLanguageFromKey('testing.testing_in_testenv_title', this.props.language)}
+              </Typography>
+              <Typography variant='body1' className={classes.ingress}>
+                {getLanguageFromKey('testing.testing_in_testenv_body', this.props.language)}
+              </Typography>
+            </Grid>
+
+            <Grid item={true} xs={11} sm={11} md={7} className={classes.deployPlaceholderStyle}>
+
+              <DeployPaper
+                titleTypographyVariant='h2'
+                localRepoInSyncWithMaster={this.returnInSyncStatus(this.props.repoStatus)}
+                cSharpCompiles={true}
+                masterRepoAndDeployInSync={this.props.deploymentList.at21.items.length > 0 ?
+                  this.returnMasterRepoAndDeployInSync('at21', this.props.masterRepoStatus, this.props.deploymentList)
+                  :
+                  null
+                }
+                deploySuccess={deploySuccess}
+                deployFailedErrorMsg='Some error'
+                language={language}
+              />
+
+            </Grid>
+            <Hidden mdUp={true} smDown={true}>
+              <Grid item={true} sm={2}>
+                {/* grid padding */}
+              </Grid>
+            </Hidden>
+
+            <Grid item={true} xs={11} sm={11} md={4} className={classes.aboutServicePlaceholderStyle}>
+
+              <Paper square={true} elevation={1} style={{ padding: 24, maxWidth: 800 }}>
+                Placeholder for "Tjenesten i testmiljø"
+              </Paper>
+
+            </Grid>
+            <Hidden mdUp={true} smDown={true}>
+              <Grid item={true} sm={3}>
+                {/* grid padding */}
+              </Grid>
+            </Hidden>
+          </Grid>
+
+        </div>
+      </React.Fragment >
+    );
+  }
+}
+
+const makeMapStateToProps = () => {
+  const mapStateToProps = (
+    state: IServiceDevelopmentState,
+  ) => {
```

Comment:
Can't see the point of wrapping mapStateToProps in the makeMapStateToProps-function.
Any reason for this?

Reply:
Agree. This is boilerplate if selectors are used. I will remove this if selectors are not used.